pb_plugins/dcsdkgen/enum.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from .name import Name
```

Comment:
Do we want to call the class `Name`? Like `plugin_name` will be a `Name` as well (which is why I tried "CaseName" in the issue, which is not ideal).